calibration/CTDMOG/CGINS-CTDMOG-13437__20161025.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13437,CC_ptempa0,1.474392e+002,
 37-13437,CC_ptempa1,-7.474336e-002,
 37-13437,CC_ptempa2,1.741181e-006,
-37-13437,CC_wbotc,2.4644e-007,
+37-13437,CC_wbotc,2.464384e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT